src/test/java/org/apache/commons/lang3/RandomUtilsTest.java

```diff
@@ -40,7 +39,7 @@
 
     @Test
     public void testConstructor() {
-        assertNotNull(new RandomUtils());
```

Comment:
The `new` operator never returns `null`. `new RandomUtils()` cannot be `null` in Java.

Reply:
If you clear the assertion, it seems useless to keep an expression for the new object.

If not used, it should be removed.

Reply:
Done. Also in the other cases and all the default constructors were removed in these cases that do not need to be there explicit. I moved the relevant part of the constructor javadoc to the class documentation.

Reply:
Oh, sorry.

I don't think I'm clear, I mean delete is to delete the useless expression like `new RandomUtils()`.

Reply:
That was absolutely clear, but I was picking up the thread when I checked why the test instantiates the class at all and then I could see that

1.) there was no reason
2.) the constructors were calling super() which is done by default
3.) the constructors were the default constructors that do not need to be written
4.) some tests just tested that the class with only the default constructor exists in a different way

so I deleted the constructors, where they are created anyway and I changed the tests to follow the same pattern.